db/changes.go

```diff
@@ -196,7 +196,7 @@ func (db *DatabaseCollectionWithUser) buildRevokedFeed(ctx context.Context, ch c
 	paginationOptions.Since.Seq = revokeFrom
 
 	// Use a bypass channel cache for revocations (CBG-1695)
-	singleChannelCache, err := db.changeCache.getChannelCache().getBypassChannelCache(ch)
+	singleChannelCache, err := db.changeCache().getChannelCache().getBypassChannelCache(ch)
```

Comment:
Since we've only got a single channel cache, it feels like getting the channel cache via the database context might be more direct.  Possibly a future refactor?